aten/src/ATen/ConjugateFallback.cpp

```diff
@@ -0,0 +1,52 @@
+#include <ATen/ConjugateFallback.h>
+#include <ATen/native/UnaryOps.h>
+
+namespace at {
+
+    TORCH_LIBRARY_IMPL(_, Conjugate, m) {
+        m.fallback(torch::CppFunction::makeFromBoxedFunction<&conjugateFallback>());
+    }
+
+    void conjugateFallback(const c10::OperatorHandle& op, torch::jit::Stack* stack) {
+        // Unwrap all arguments
+        const auto num_arguments = op.schema().arguments().size();
+        const auto arguments = torch::jit::last(stack, num_arguments);
```

Comment:
`torch::jit::last` returns a non-owning reference to the arguments. When you pop them later in line 14, this reference is no longer valid; therefore your memory corruption.